src/sentry/tasks/post_process.py

```diff
@@ -87,7 +87,7 @@ def handle_owner_assignment(project, group, event):
     from sentry.models import GroupAssignee, ProjectOwnership
 
     with metrics.timer("post_process.handle_owner_assignment"):
-        owners_ingestion = features.has("organizations:workflow-owners", event.project)
+        owners_ingestion = features.has("organizations:workflow-owners", event.project.organization)
```

Comment:
Has this organization been eagerly loaded? This could add an additional query which can't go out without caching.

Reply:
On line 248, before `handle_owner_assignment` is called, we have `bind_organization_context(event.project.organization)` which I think does this query and caches it for future use.